core/point/point.go

```diff
@@ -1,13 +1,25 @@
 // Package point is a basic package for points.
 package point
 
+import (
+	"bytes"
+)
+
 // Point is a basic point. Although simple, the member variables are kept
 // private to ensure that Point remains immutable.
 type Point struct {
 	x int64
 	y int64
 }
 
+// PointSGFSlce is a slice/Array translation reference between integer
+// -Point (index position) and string SGF-Point (byte/char) values
+var PointSGFSlce = []byte{'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h',
```

Comment:
How about you this a map so you don't have to scan?

ex: map[rune]int{
'a': 1
'b': 2
...
}

Reply:
That is a good idea. A map would be more efficient. However, I'm not sure in GoLang how to reverse map, ie retrieve a key from a value, as when translating from Point to SGF and vice versa. Hence, I made two maps. One has the rune/char values as keys, and one has the int64 values as keys. Also, I left the whole thing 0 indexed as we had previously discussed, and because non-zero based seemed to have some issues and require pushing the translations +1 or -1 to make them work. Let me know if I am missing something. I will be submitting a pull request shortly.

Reply:
Actually, my update simply updated this pull request. The changes can be seen in the files changed section. Thanks!